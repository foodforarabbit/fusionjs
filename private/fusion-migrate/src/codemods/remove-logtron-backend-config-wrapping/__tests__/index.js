const path = require('path');
const pluginTester = require('babel-plugin-tester');

pluginTester({
  plugin: require('../plugin'),
  fixtures: path.join(__dirname, '../__fixtures__'),
});
