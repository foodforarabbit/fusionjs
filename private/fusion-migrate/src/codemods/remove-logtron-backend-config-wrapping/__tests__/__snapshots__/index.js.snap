// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`remove-logtron-backend-config-wrapping positive-match: positive-match 1`] = `
"app.register(LogtronBackendsToken, {backends: {foo: bar}});


      ↓ ↓ ↓ ↓ ↓ ↓

app.register(LogtronBackendsToken, {
  foo: bar,
});
"
`;
